{{cookiecutter.project_slug}}/ansible/templates/supervisord.conf.j2

```diff
@@ -29,3 +29,13 @@ priority = 10
 redirect_stderr = false
 autostart = true
 autorestart = true
+
```

Comment:
Hier kan je een mooie truc gebruiken om onderscheid tussen web, taak en staging te maken:
```
{% if 'gunicorn' in supervisor_programs %}
[program:gunicorn] ...
{% endif %}

{% if 'celery' in supervisor_programs %}
[program:celeryd] ...
{% endif %}
```

En vervolgens in je inventories:
```
[web:vars]
gunicorn_port = 5050
gunicorn_workers = 3
gunicorn_max_requests = 500
supervisor_programs: [
    'gunicorn',
]
[task:vars]
celery_concurrency = 1
supervisor_programs: [
    'celery',
]
```

Op staging kan je die samenvoegen:
```
supervisor_programs: [
    'gunicorn',
    'celery',
]
```